core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1186,8 +1184,9 @@ public VersionNumber getNeededDependenciesMinimumJavaVersion() {
             return versionNumber;
         }
 
-        private void logBadMinJavaVersion(String s, String pluginName, String minimumJavaVersion) {
-            LOGGER.log(Level.WARNING, s, new String[]{pluginName, minimumJavaVersion});
+        private void logBadMinJavaVersion(String minimumJavaVersion) {
```

Comment:
As the calls to this method is always the same you can even avoid passing the **minimumJavaVersion** argument.

Reply:
:+1: done.